src/Symfony/Component/Console/Helper/TableStyle.php

```diff
@@ -46,6 +46,8 @@ class TableStyle
     private string $cellRowFormat = '%s';
     private string $cellRowContentFormat = ' %s ';
     private string $borderFormat = '%s';
+    private bool $intro = true;
```

Comment:
I don't know what is a practice in Symfony to name boolean properties, but IMO it's more clear when they have `is*` or `has*` prefixes.

Reply:
good point. Also I was thinking to only use one field "$hasFraming" and not allow setting intro and outro separately. Also I am not sure if "framing" is a good name.
Thoughts?